fsw/tables/sc_rts002.c

```diff
@@ -29,9 +29,9 @@
  * This source file creates a sample RTS table that contains only
  * the following commands that are scheduled as follows:
  *
- * SC NOOP command, execution time relative to start of RTS = 0
- * SC NOOP command, execution time relative to prev cmd = 5
- * SC NOOP command, execution time relative to prev cmd = 5
+ * SC NOOP command, execution wakeup count relative to start of RTS = 0
+ * SC NOOP command, execution wakeup count relative to prev cmd = 1
+ * SC NOOP command, execution wakeup count relative to prev cmd = 1
```

Comment:
Isn't relative to previous cmd wakeup count = 5 still for both?

Reply:
Sorry, yes - have changed it back to 5